drivers/SmartThings/zigbee-button/src/zigbee-multi-button/supported_values.lua

```diff
@@ -25,6 +25,7 @@ local devices = {
       { mfr = "CentraLite", model = "3450-L" },
       { mfr = "CentraLite", model = "3450-L2" },
       { mfr = "ROBB smarrt", model = "ROB_200-008-0" }
+      
```

Comment:
Can we please remove this extra line that is empty?

Reply:
Remove the extra line.